databox/api/src/Metadata/MetadataNormalizer.php

```diff
@@ -47,6 +47,7 @@ public static function createBlankMeta(): array
         return [
             'name' => null,
             'values' => [],
+            'value' => null,
```

Comment:
should be "" to protect twig, but anyway empty value(s) are **now** removed before render.
todo (?): filtering empty value(s) may prevent to initialize an attribute with **real** empty value from metadata.
So let's keep null as discriminant, and fix filtering to preserve real empty string.

Reply:
Need to be revisited in general, cf my previous comment